pkg/authorization/authorizer/authorizer_test.go

```diff
@@ -308,6 +344,18 @@ func newDefaultGlobalPolicy() ([]authorizationapi.Policy, []authorizationapi.Pol
 						// until we get components authenticating, mssing users will be given all rights.  Yay, security!
 						UserNames: []string{"ClusterAdmin"},
 					},
+					"user-only": {
+						ObjectMeta: kapi.ObjectMeta{
+							Name:      "user-only",
+							Namespace: testMasterNamespace,
+						},
+						RoleRef: kapi.ObjectReference{
+							Name:      "basic-user",
+							Namespace: testMasterNamespace,
+						},
+						// until we get components authenticating, mssing users will be given all rights.  Yay, security!
```

Comment:
pulled the stale comment.  That creates a binding that allows us to test resourceNames using the bootstrap policy.